Source/ablastr/parallelization/MPIInitHelpers.cpp

```diff
@@ -20,6 +20,11 @@
 #include <hip/hip_runtime.h>
 #endif
 
+// INC0509050: Handle MPI initialization/finalization errors on Tuolumne (LLNL)
+#if defined(AMREX_USE_HIP)
```

Comment:
Is this the right macro or should it be `AMREX_USE_ROCTX`?

Reply:
This is the one we want.